lib/elixir_analyzer/test_suite/strain.ex

```diff
@@ -0,0 +1,37 @@
+defmodule ElixirAnalyzer.TestSuite.Strain do
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise Strain
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+
+  assert_no_call "does not call Enum.filter/2" do
+    type :essential
+    called_fn module: Enum, name: :filter
+    comment ElixirAnalyzer.Constants.strain_use_recursion()
```

Comment:
If this gets merged after #163, the comments can be called with an alias `Constants.strain_...`.